environment.yml

```diff
@@ -3,4 +3,5 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - gdal
+  - python=3.8
+  - rasterio>=1
```

Comment:
See https://github.com/stactools-packages/aster/pull/4/files for some backflips to get `rasterio` builds working through pip. Per https://github.com/stac-utils/stactools/issues/144, our conda vs. pip story a bit confused. I think for subpackages it's ok to do things a little different, but IMO if the build instructions don't need `conda`, then we should try to keep it just to pip. Did you need a `conda` env to make tests work locally?

Reply:
i'm just accustomed to using conda for virutal environments... could add alternative docs later for just pip